en/docs/references/adaptive-authentication-js-api-reference.md

```diff
@@ -366,19 +362,19 @@ the following parameters.
 </thead>
 <tbody>
 <tr class="odd">
-<td>name</td>
-<td>Name of the cookie.</td>
+<td>Request</td>
+<td>HTTP authentication request.</td>
 </tr>
 <tr class="even">
-<td>value</td>
-<td>Value of the cookie.</td>
+<td>name</td>
+<td>Name of the cookie.</td>
 </tr>
 <tr class="odd">
 <td>properties</td>
-<td><p>A map that may contain optional attributes of the cookie ‘ <code>                             de                           </code> <code>                             crypt                           </code> <strong>validateSignature</strong></p>
+<td>A map that may contain optional attributes of the cookie <code>decrypt</code> and <code>validateSignature</code>
```

Comment:
Missing fullstop

Reply:
fixed in https://github.com/wso2/docs-is/pull/2617